tests/openassetio/test_logging.py

```diff
@@ -63,28 +63,30 @@ def severity_filter(mock_logger):
     return lg.SeverityFilter(mock_logger)
 
 
-class TestSeverityFilter():
-    all_severities = (
-        lg.LoggerInterface.kDebugAPI,
-        lg.LoggerInterface.kDebug,
-        lg.LoggerInterface.kInfo,
-        lg.LoggerInterface.kProgress,
-        lg.LoggerInterface.kWarning,
-        lg.LoggerInterface.kError,
-        lg.LoggerInterface.kCritical
-    )
+all_severities = (
+    lg.LoggerInterface.kDebugAPI,
+    lg.LoggerInterface.kDebug,
+    lg.LoggerInterface.kInfo,
+    lg.LoggerInterface.kProgress,
+    lg.LoggerInterface.kWarning,
+    lg.LoggerInterface.kError,
+    lg.LoggerInterface.kCritical
+)
 
-    def test_constructor(self, mock_logger, monkeypatch):
+severity_control_envvar = "OPENASSETIO_LOGGING_SEVERITY"
 
-        var = "OPENASSETIO_LOGGING_SEVERITY"
 
-        # Unset
+class TestSeverityFilter_constructor():
```

Comment:
[consider][minor] See other comment on `Test_Session_Constructor` re. how we should name constructor tests. Regardless what we choose, its nice to have a consistent style (in this case `_Constructor` vs `_constructor`).

Reply:
Updated [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/214/commits/2e3b34fd17b53f1924a20b5ab4cb852566a13c48).